.eslintignore

```diff
@@ -5,3 +5,5 @@ libraries/botbuilder-dialog-prompts/**/*
 libraries/botbuilder-spell-check-middleware/**/*
 libraries/botbuilder-storage/**/*
 libraries/botbuilder-text-analytics-middleware/**/*
+libraries/botbuilder-middleware-text-analytics/**/*
```

Comment:
Why are both packages added to ESLint ignore? Or will we fix this in the v1 release?

Reply:
This package was well into development before the eslint stuff was added, so my attention is to get it into the repo first, and then fix the eslint as a separate PR. I did run the eslint-fix for them, which made _some_ modifications.

Reply:
👍